python/gsw/main.py

```diff
@@ -0,0 +1,350 @@
+from flask import Flask, request, jsonify
+from flasgger import Swagger
+from flasgger.utils import swag_from
+from elasticsearch import Elasticsearch
```

Comment:
Can you:
- Create a python venv for running Ground Software
- Add all these items to a requirements.txt file using pip? `pip freeze > requirements.txt`

It would be nice to also add a README so that everyone can set up ground software with a common set of instructions.